### ISSUE TITLE
Does this Splunk app work with Security Onion?

### ISSUE THREAD

**Issue by bradpoulton** [jahshuah/splunk-ta-bro-json#2]
Can i use this app with Security onion 16.04?

**Comment by bradpoulton** [comment]
Yes, it does work at properly parsing Bro logs on Security onion. 

inputs.conf
`[monitor:///nsm/bro/logs/current]
index = bro
sourcetype = json_bro
whitelist = .*\.log$
blacklist = .*(communication|stderr)\.log$
disabled = 0
`

**Comment by kneecapped** [comment]
Were you able to get the alias to properly change to id.orig_h to src/src_ip . Mine seems to be having issues.

**Comment by bradpoulton** [comment]
I noticed the same issue in my setup and then realized the app developer intends for the app to be installed on the Universal Forwarder, Indexer, and Search Head. I only installed in on the UF, which gave me the logs and the SH auto extracted the JSON, but without the app on the Indexer and Search Head the built in field aliases will not work. I will open a another ticket and suggest the readme is updated for this git project. This app really need to be in the Splunk apps website.
